src/app.service.js

```diff
@@ -290,10 +290,19 @@ export class AppService {
     };
   }
 
-  getOpenGraph(property, headElement) {
-    return headElement.querySelector(
-      `meta[property='og:${property}'], meta[name='${property}']`,
-    )?.content;
+  getOpenGraph(properties, headElement) {
+    if (!Array.isArray(properties)) {
+      properties = [properties];
+    }
+
+    return properties
+      .map(
+        (property) =>
+          headElement.querySelector(
+            `meta[property='og:${property}'], meta[name='${property}']`,
+          )?.content,
+      )
+      .find((content) => content !== undefined);
   }
```

Comment:
```suggestion
  getOpenGraph(headElement, ...properties) {
    return properties
      .map(
        (property) =>
          headElement.querySelector(
            `meta[property='og:${property}'], meta[name='${property}']`,
          )?.content,
      )
      .find((content) => content !== undefined);
  }
```



`const title = this.getOpenGraph(headElement, "title") || headElement.querySelector("title")?.textContent;`
`const siteName = this.getOpenGraph(headElement, "site_name", "article:author");`

이런 방법도 있을 것 같습니다~!
수정하실 필요는 없습니다.